File: generated-sources/scala-gatling/mojang-sessions/src/gatling/scala/org/openapitools/client/model/PlayerTextureURL.scala

package org.openapitools.client.model


case class PlayerTextureURL (
    /* The URL to the texture, must be in Mojang's domains. */
    _url: String
)
object PlayerTextureURL {
    def toStringBody(var_url: Object) =
        s"""
        | {
        | "url":$var_url
        | }
        """.stripMargin
}

File: generated-sources/php-ze-ph/mojang-sessions/src/App/com/github/asyncmc/mojang/sessions/php/ze/ph/model/PlayerTextureURL.php
<?php

namespace com.github.asyncmc.mojang.sessions.php.ze.ph.server\com.github.asyncmc.mojang.sessions.php.ze.ph.model;

use Articus\DataTransfer\Annotation as DTA;

/**
 * Wraps the URL to the texture
 */
class PlayerTextureURL
{
    /**
     * The URL to the texture, must be in Mojang&#39;s domains.
     * @DTA\Data(field="url")
     * @DTA\Validator(name="Type", options={"type":"string"})
     * @var string
     */
    public $url;
}
